tools/ci_fetch_deps.py

```diff
@@ -3,10 +3,45 @@
 import time
 import shlex
 import pathlib
+import re
 import subprocess
 
-# Target will be a board, "test", "docs", "mpy-cross-mac", or "windows"
-TARGET = sys.argv[1]
+TOP = pathlib.Path(__file__).parent.parent
+
+
+def _git_version():
+    version_str = subprocess.check_output(["git", "--version"], encoding="ascii", errors="replace")
+    version_str = re.search("([0-9]\.*)*[0-9]", version_str).group(0)
+    return tuple(int(part) for part in version_str.split("."))
+
+
+clone_supports_filter = (
+    False if "NO_USE_CLONE_FILTER" in os.environ else _git_version() >= (2, 27, 0)
```

Comment:
I didn't catch this: I thought the breakpoint was at 2.36.0 for partial clones, not 2.27.